### ISSUE TITLE
Any bug on RACE?

### ISSUE THREAD

**Issue by YeDeming** [StonyBrookNLP/deformer#9]
Hi Qingqing,

Thanks for releasing great code. 

I have successfully trained sbert for SQuAD.  But when I train original bert on RACE, the loss can not convergence even the number of training dataset is 2 (=1 is ok).  Is there any bug about the RACE dataset?

If it is convenient, could you please provide me serveral sbert for RACE, such as sbert-s7/sbert-s9/sbert-s10?

Thanks, Looking forward to your reply!

Best, 
Deming

**Comment by YeDeming** [comment]
After I shuffle the train.jsonl mannualy, it works.
